source/GameLoadingPanel.cpp

```diff
@@ -108,4 +119,12 @@ void GameLoadingPanel::Draw()
 		a += da;
 	}
 	PointerShader::Unbind();
+
```

Comment:
```suggestion

	hint = loadingHintArray[Random::Int(sizeof(loadingHintArray))];
```

Reply:
Doing it this way leads to the hint being re-chosen every time the loading screen is drawn, but I've moved it to GameLoadingPanel::GameLoadingPanel(), where I think it works okay.

Reply:
oh, I wanted to place it there, got the wrong line